modules/ml/preprocessor/base.py

```diff
@@ -4,7 +4,8 @@
 class BasePreProcessor:
     def process(self, document: dict) -> List[dict]:
         """
-        Perform document cleaning and splitting. Takes a single document as input and returns a list of documents.
+        Perform document cleaning and splitting.
```

Comment:
One blank line between the first line and  the rest.
```suggestion
        """Performs document cleaning and splitting.
        
        Takes a single document as input and returns a list of documents.
```